framework/src/modules/interoperability/constants.ts

```diff
@@ -69,11 +69,23 @@ export const CCM_STATUS_MODULE_NOT_SUPPORTED = 1;
 export const CCM_STATUS_CROSS_CHAIN_COMMAND_NOT_SUPPORTED = 2;
 export const CCM_STATUS_CHANNEL_UNAVAILABLE = 3;
 export const CCM_STATUS_CODE_FAILED_CCM = 4;
-export const CCM_STATUS_CODE_CHANNEL_UNAVAILABLE = 1;
+export const CCM_PROCESSED_CODE_SUCCESS = 0;
 export const CCM_PROCESSED_CODE_CHANNEL_UNAVAILABLE = 1;
+export const CCM_PROCESSED_CODE_MODULE_NOT_SUPPORTED = 2;
+export const CCM_PROCESSED_CODE_CROSS_CHAIN_COMMAND_NOT_SUPPORTED = 3;
+export const CCM_PROCESSED_CODE_FAILED_CCM = 4;
+export const CCM_PROCESSED_CODE_INVALID_CCM_VALIDATION_EXCEPTION = 5;
+export const CCM_PROCESSED_CODE_INVALID_CCM_VERIFY_CCM_EXCEPTION = 6;
+export const CCM_PROCESSED_CODE_INVALID_CCM_VERIFY_EXCEPTION = 7;
+export const CCM_PROCESSED_CODE_INVALID_CCM_BEFORE_CCC_EXECUTION_EXCEPTION = 8;
+export const CCM_PROCESSED_CODE_INVALID_CCM_AFTER_CCC_EXECUTION_EXCEPTION = 9;
+export const CCM_PROCESSED_CODE_INVALID_CCM_BEFORE_CCC_FORWARDING_EXCEPTION = 10;
```

Comment:
maybe we can have enum here named `CCM_PROCESSED_CODE` for all these codes

Reply:
yea, i wanted to convert all these to the enum, but i thought we have different issue for that